lib/typing/ext/variables.go

```diff
@@ -44,12 +44,17 @@ var SupportedTimeFormats = []string{
 	AdditionalTimeFormat,
 }
 
+const TimezoneOffset = "Z07:00"
```

Comment:
Nit: want to call this `TimezoneOffsetFormat`? So it's clear that it's just being used as a format string and not a hardcoded offset that we're using for anything.